Apollo/ApolloExtra.lua

```diff
@@ -3148,14 +3148,14 @@ if ModUtil ~= nil then
 				HasTraitNameInRoom = "MasterBoltTrait",
 				{ Cue = "/VO/Apollo_0041",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort",
-					Text = "Seeing you struggle with your father made me realise something, Zagzag. Fathers and sons work much better together. My father and I put together something that might help you." },
```

Comment:
I'll assume all changes here are the same as the one you ask vivid to re-take. I'll accept those changes, but I'll verify the recording to make sure it's the same sentences.

Reply:
They are! Others are corrected slightly to be accurate to how he actually reads the lines (as opposed to precisely how they were originally written); I corrected them while listening line-by-line to the audio extracted from the VO.fsb